fe/fe-core/src/main/java/com/starrocks/sql/parser/SqlParser.java

```diff
@@ -50,6 +50,7 @@
 public class SqlParser {
     private static final Logger LOG = LogManager.getLogger(SqlParser.class);
     private static final String EOF = "<EOF>";
+    private static final int MIN_TOKEN_LIMIT = 100;
```

Comment:
this may affect some tests.  Please consider add a check when set this session variable?